<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>831</width>
      <height>461</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Ajouter">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QPushButton" name="verfier">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>verifer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>30</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de téléphone du propriétére</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="a12_1">
         <property name="text">
          <string>Oui</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="a12_2">
         <property name="text">
          <string>Non</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="enregistrer">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>enregistrer</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="v">
         <property name="text">
          <string>A vendre</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="l">
         <property name="text">
          <string>A louer</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>456</width>
        <height>472</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="a7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="a8"/>
       </item>
       <item>
        <widget class="QLineEdit" name="a9"/>
       </item>
       <item>
        <widget class="QLineEdit" name="a10"/>
       </item>
       <item>
        <widget class="QLineEdit" name="a11"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>101</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="a1">
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="a2">
         <property name="text">
          <string>Piéce</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="a3">
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="a4">
         <property name="text">
          <string>Ville</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="a5">
         <property name="text">
          <string>Coss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="a6">
         <property name="text">
          <string>Meublé</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>150</y>
        <width>121</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="b7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="b8"/>
       </item>
       <item>
        <widget class="QLineEdit" name="b9"/>
       </item>
       <item>
        <widget class="QLineEdit" name="b10"/>
       </item>
       <item>
        <widget class="QLineEdit" name="b11"/>
       </item>
       <item>
        <widget class="QLineEdit" name="b12"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>111</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="b1">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="b2">
         <property name="text">
          <string>Preonm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="b3">
         <property name="text">
          <string>Tel1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="b4">
         <property name="text">
          <string>Tel2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="b5">
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="b6">
         <property name="text">
          <string>Code Postale</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimer">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLineEdit" name="c">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>541</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="client">
         <property name="text">
          <string>Client</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="prop">
         <property name="text">
          <string>Proprietaire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="appart">
         <property name="text">
          <string>Appartement</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="a15">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tel :</string>
      </property>
     </widget>
     <widget class="QLabel" name="a16">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="a12_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Untitled-1.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Louer">
     <attribute name="title">
      <string>Louer</string>
     </attribute>
     <widget class="QLineEdit" name="p1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="v1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numap">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID de l'appart</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="champ">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>521</width>
        <height>191</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
     </widget>
     <widget class="QPushButton" name="valider">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner num tel client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numtel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>471</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="p">
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ville">
         <property name="text">
          <string>Ville</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="s">
         <property name="text">
          <string>s + ?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Vendre">
     <attribute name="title">
      <string>Vendre</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>471</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="p_2">
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ville_2">
         <property name="text">
          <string>Ville</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="s_2">
         <property name="text">
          <string>s + ?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="s1_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID de l'appart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="champ_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>521</width>
        <height>191</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Donner num tel client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="v1_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="numap_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="numtel_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>390</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="p1_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="listage">
     <attribute name="title">
      <string>Listage</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>481</width>
        <height>341</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ForbiddenCursor</cursorShape>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>541</width>
        <height>20</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>prop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>alouer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>client</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>bail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>avendre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string>Visite</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
